conda-forge.yml

```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+azure:
+  free_disk_space: True
+  upload_packages: True
```

Comment:
This set to `True` by default. Do we need it set here?

Reply:
:upside_down_face: This was me thinking that was the artifact upload, oops!